remoulade/message.py

```diff
@@ -116,6 +119,17 @@ def cancel(self):
         backend = broker.get_cancel_backend()
         backend.cancel([self.message_id])
 
+    def set_progress(self, progress):
+        """ Set the progress of the message.
+            progress(float) number between 0 and 1 inclusive
```

Comment:
add a `Raises` entry to the docstring, look around in the code there is plenty of them. it raise `NoStateBackend` or `InvalidProgress`.